### PR TITLE
1.5.17 release

### PR DESCRIPTION
**Pull request by goeffthomas**

### CODE REVIEW

File: kaggle/api/kaggle_api_extended.py [hunk 1]
```diff
@@ -323,8 +323,8 @@ class KaggleApi(KaggleApi):
     ]
 
     # Datasets valid types
-    valid_dataset_file_types = ['all', 'csv', 'sqlite', 'json', 'bigQuery']
-    valid_dataset_license_names = ['all', 'cc', 'gpl', 'odb', 'other']
+    valid_dataset_file_types = ['unspecified', 'csv', 'sqlite', 'json', 'bigQuery']
```
Comment (goeffthomas) [inline kaggle/api/kaggle_api_extended.py:326]: Alternatively, we could leave these as `'all'` to keep the values uniform across Comps/Notebooks. Then on 1152-1153, do a simple conversion from `'all'` to `'unspecified'`.

**Review by Philmod — COMMENTED** [review]
I do prefer https://github.com/Kaggle/kaggleazure/pull/25838 indeed

**Review by jeradrose — APPROVED** [review]
LGTM! Sorry for the churn caused by my PR.

**Comment by goeffthomas** [comment]
Closing this PR and going with the MT-based approach.

**Pull request closed without merging**